src/utils/chain.ts

```diff
@@ -11,12 +11,15 @@
  *
  * Removal or modification of this copyright notice is prohibited.
  */
-
 import cli from 'cli-ux';
 import { Block, BlockHeader } from '@liskhq/lisk-chain';
 import { getAPIClient } from '../client';
+import { NETWORKS } from '../constants';
+import { NodeInfo } from '../types';
 
 let blockIDAtSnapshotHeight: string;
+let TOKEN_ID_LSK: string;
+let HEIGHT_PREVIOUS_SNAPSHOT_BLOCK: number;
```

Comment:
Please use camel-case for `let` variables.
```suggestion
let tokenIDLsk: string;
let heightPreviousSnapshotBlock: number;
```